### PR TITLE
Mobile formatting

### PR DESCRIPTION
**Pull request by maidiance**
## Description

We want our webpage to have mobile formatting as per Figma design. This PR focuses on the Landing Page.

Co-authored-by: [Javier Rosado](https://github.com/jrosado184)

#### Video Link

[Loom Video](https://www.loom.com/share/a61db5c86b1049f3be81edf18fe3222a)

#### Trello Link

<blockquote class="trello-card"><a href="https://trello.com/c/2Lhsm3dI/288-as-a-user-i-would-like-the-website-to-be-mobile-friendly">As a user, I would like the website to be mobile friendly.</a></blockquote>
Note: Changed focus specifically for landing page for this pull request.

## Type of change

- [✔️] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [✔️] My code follows the style guidelines of this project
- [✔️] I have performed a self-review of my own code
- [✔️] I have removed unnecessary comments/console logs from my code
- [✔️] My changes generate no new warnings
- [✔️] I have checked my code and corrected any misspellings
- [✔️] No duplicate code left within changed files
- [✔️] Size of pull request kept to a minimum
- [✔️] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by acoman-cloud — APPROVED** [review]
This looks good. Cleaning up the CSS and making mobile look better.

**Review by hollberg — APPROVED** [review]
Looks really great in mobile view. Wonderful work!

**Review by jcpcab — APPROVED** [review]
Great Work! Way to go making the app more accessible and maintaining cleaniness

File: src/components/pages/LandingPage/LandingPage.css [hunk 1]
```diff
@@ -48,21 +51,17 @@
...
 }
-.learn {
-  height: 40px;
-  width: 130px;
-  background-color: white;
-  border: 1px solid #003d71;
-  color: #003d71;
-  border-radius: 5px;
-}
+.learn,
```
Comment (BrittniKarat) [inline src/components/pages/LandingPage/LandingPage.css:64]: Great job, keeping the code DRY here.

**Review by BryceSlade — APPROVED** [review]
Excellent responsive design with clean, easy to read code. Important addition to the application!

File: src/components/pages/FooterLanding/FooterStyle.js [hunk 2]
```diff
@@ -20,14 +20,25 @@ export const FooterStyle = styled.div`
   }
   @media only screen and (max-width: 768px) {
     .container {
-      flex-direction: column;
+      display: flex;
```
Comment (morgankj) [inline src/components/pages/FooterLanding/FooterStyle.js:23]: Always a fan of Flex! Great option for scalability

**Review by morgankj — APPROVED** [review]
Great job as always! Removed unnecessary blocks while maintaining DRY code! Mobile design cannot be overlooked!

**Pull request merged**